src/components/ToggleButton.js

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes} from 'react'
+import styles from './ToggleButton.css'
+import CSSModules from 'react-css-modules'
+import {changeTokenType} from '../actions/actions.js'
+import getID from '../utils/getID'
+
+class ToggleButton extends React.Component {
+  componentWillMount () {
+    this.state = {isChecked: false}
+    this.onChange = this.onChange.bind(this)
+    this.toggleId = getID()
+  }
+  onChange (e) {
+    let nextVal = !this.state.isChecked
+    this.setState({
+      isChecked: nextVal
+    })
+    // we need to inverse the nextVal because checked means isPreview === false
+    this.props.changeHandler(!nextVal)
+    changeTokenType(!nextVal)
```

Comment:
For the same reason as above we shouldn't have dependency on `changeTokenType` action here. It should be used in `changeHandler` that is passed to this component by its parent (https://github.com/contentful/discovery-app-react/pull/3/files#diff-40ffb4094e810bef589a1e0c4fe5dd4aR14).

What is more we should call handler with a current value, not a negation (so argument of `changeHandler` should be `isProduction`).